### PR TITLE
Switch to usethis standalone format

### PR DESCRIPTION
**Pull request by lionel-**
So that you can do this to update files, pulling in the depedencies automatically (the `ref = ` is needed until https://github.com/r-lib/usethis/pull/1821 is merged):

```r
usethis::use_standalone("r-lib/processx", "errors", ref = "standalone")
```

### CODE REVIEW

File: R/aaa-standalone-rstudio-detect.R [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# ---
+# repo: r-lib/processx
```
Comment (gaborcsardi) [inline R/aaa-standalone-rstudio-detect.R:2]: The canonical location of this is in cli, actually.

File: R/standalone-errors.R [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-# If can allow package dependencies, then you are probably better off
-# using rlang's functions for errors.
-#
-# The canonical location of this file is in the processx package:
-# https://github.com/r-lib/processx/blob/main/R/errors.R
+# ---
+# repo: r-lib/processx
+# file: standalone-errors.R
+# last-updated: 2023-01-15
+# license: https://unlicense.org
```
Comment (gaborcsardi) [inline R/standalone-errors.R:5]: Why do we need to change the license? I think it is simpler and also nicer to users to keep the same license as the licence of the package itself.

**Review by gaborcsardi — COMMENTED** [review]
Thanks!

File: R/aaa-standalone-rstudio-detect.R (hunk 1 above)
Comment (lionel-) [reply R/aaa-standalone-rstudio-detect.R:2]: Now done in https://github.com/r-lib/usethis/pull/1821/commits/4d32858d53641ec55704cb3ecf21b99bc1657149

But in cli there is also this mention:

```r
# The canonical location of this file is in the processx package:
# https://github.com/r-lib/processx/blob/main/R/errors.R
```

And the last commit in the cli errors file says:

```
Update errors.R from processx
```

Comment (lionel-) [reply R/aaa-standalone-rstudio-detect.R:2]: Conversation on Slack clarified that it's only the rstudio file that's from cli. Also it's a soft dependency so the errors file doesn't need to depend on it.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**